spec/examples/metal/keyspace_spec.rb

```diff
@@ -86,6 +86,28 @@
     end
   end
 
+  describe "#configure" do
```

Comment:
@ryakh This looks fine to me. I'm wondering if you can create a small test that pushes some data back and forth in both of these cases, so we all can be sure it works?

Reply:
+1